ui/src/app/utils/access-utils.tsx

```diff
@@ -158,18 +145,6 @@ export const getRegistrationTasks = (navigate): RegistrationTask[] => serverConf
     description: <span>Sign the Data User Code of Conduct consenting to the <AoU/> data use policy.</span>,
     buttonText: 'View & Sign',
     completedText: 'Signed',
-    completionTimestamp: (profile: Profile) => {
```

Comment:
This logic is now enforced by API Offline Sync: the DUCC Completion time is removed if the version does not match.